tools/tools.go

```diff
@@ -0,0 +1,23 @@
+// +build tools
+
+// To install the following tools at the version used by this repo run:
+// $ make tools
+// or
+// $ go generate -tags tools tools/tools.go
+
+package tools
+
+//go:generate go install github.com/hashicorp/go-bindata
+import _ "github.com/hashicorp/go-bindata"
```

Comment:
This isn't the `go-bindata` version we use. The version we use in our release builds is `github.com/kevinburke/go-bindata` https://github.com/hashicorp/waypoint/blob/da80898a086a3fc8a4f19b7180d7d198927e5355/.circleci/config.yml#L160

Reply:
Oooh yeah, I remember this now. Thanks!